ls1intum/Artemis
# Enhancement/programming online editor UI

### Checklist
- [x] Client: I added multiple integration tests related to the features (with a high test coverage)

### Motivation and Context
This is another follow-up PR concerning the online editor user interface for programming exercises.
(These are the previous ones:  https://github.com/ls1intum/Artemis/pull/3843 & https://github.com/ls1intum/Artemis/pull/3832)

This PR implements 2 changes :

1) Hides the draggable icons "||" between the left/right and middle window. These have been shown if the window (e.g. problem statement) was minimized. You could and should not be able to change the size of the minimized window.

2) Changes the look of the minimized file browser to be similar to the problem-statement/instruction window.
<image src="https://user-images.githubusercontent.com/65814168/127891864-34f4ba4e-7aab-4314-9c16-59bc7a1844a9.png" 
alt="before" width="250"/>
<image src="https://user-images.githubusercontent.com/65814168/127892721-478f2239-7b05-4183-95f8-18b90a349057.png" 
alt="after" width="250"/>




_3) additionally added the respective icon to the minimized view (requested by Stephan)_

### Description

Analog changes/implementation to the previous PRs.

### Steps for Testing

Find or create a programming exercise.
Open the online code editor.
Check if the file browser's new look works and is similar to the problem-statement/instruction window.

P.S. @SimonEntholzer : Thank you very much for helping me to install the local programming setup. 🐧👍
